.github/workflows/neuron-ci.yml

```diff
@@ -0,0 +1,245 @@
+name: NEURON CI
+
+on:
+  push:
+    branches: 
+      - master
+      - live-debug*
```

Comment:
live-debug needs to be here?

Reply:
yes, for the debug setup to be taken into account (explained in the file towards the end).

Reply:
so the CI will run for pushes to master (like travis) or for branches that start with `live-debug`